crates/environ/src/cache/config.rs

```diff
@@ -193,7 +193,7 @@ lazy_static! {
         ProjectDirs::from("", "BytecodeAlliance", "wasmtime");
     static ref DEFAULT_CONFIG_PATH: Result<PathBuf, String> = PROJECT_DIRS
         .as_ref()
-        .map(|proj_dirs| proj_dirs.config_dir().join("wasmtime-cache-config.toml"))
+        .map(|proj_dirs| proj_dirs.config_dir().join("config.toml"))
```

Comment:
Could I perhaps throw my hat in the ring for `wasmtime.toml` instead of `config.toml`?

The name `config.toml` seems sort of bland and easily confused with other tools, but I may also be misunderstanding this. Is this already located in a wasmtime-specific location? (in which case `config.toml` sounds good)

Reply:
The default directory is `<platform-specific-settings-directory>/BytecodeAlliance/wasmtime` so I felt that the generic `config.toml` file would suffice.